packages/run-it/src/components/PerfTracker/PerfTracker.tsx

```diff
@@ -43,11 +43,12 @@ import { PerfTable } from './PerfTable'
 interface PerfTrackerProps {
   perf?: PerfTimings
   showAllColumns?: boolean
+  configurator: RunItConfigurator
 }
 
-const perfFilter = (all = false) => {
+const perfFilter = (configurator: RunItConfigurator, all = false) => {
```

Comment:
Maybe this could exist in `perfUtils.tsx` instead. Might also want to add a short doc comment.

Reply:
Perhaps. Initially I had moved it inside the component. We'll have to see where it's appropriate.